SpywareFilter/sections/tracking_servers.txt

```diff
@@ -52,6 +52,7 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||leadsrx.com^$third-party
```

Comment:
Their `login` page will be broken by our DNS.
I guess we can exchange the rule to
```
||app.leadsrx.com^
```